src/legacy/core_plugins/data/public/filter/filter_bar/filter_editor/index.tsx

```diff
@@ -441,17 +441,20 @@ class FilterEditorUI extends Component<Props, State> {
       queryDsl,
     } = this.state;
 
-    const { store } = this.props.filter.$state;
+    const { $state } = this.props.filter;
+    if (!$state || !$state.store) {
```

Comment:
could below functions (buildCustomFilter/buildFilter) work without passing $state.store ?

Reply:
Out of scope for this PR